tools/run_tests/run_node.sh

```diff
@@ -48,8 +48,8 @@ then
   cd build
   gcov Release/obj.target/grpc/ext/*.o
   lcov --base-directory . --directory . -c -o coverage.info
-  genhtml -o ../ext_coverage --num-spaces 2 -t 'Node gRPC test coverage' \
-    coverage.info
+  genhtml -o ../../../reports/node_ext_coverage --num-spaces 2 \
+    -t 'Node gRPC test coverage' coverage.info
```

Comment:
Thanks :-) I'm merely doing some Jenkins tests for now, but I'll change that too.